mixer/cmd/build.go

```diff
@@ -585,6 +592,12 @@ func init() {
 
 	RootCmd.AddCommand(buildCmd)
 
+	buildAllCmd.Flags().IntVar(&buildFlags.downloadRetries, "retries", retriesDefault, "Number of retry attempts to download RPMs")
+	buildBundlesCmd.Flags().IntVar(&buildFlags.downloadRetries, "retries", retriesDefault, "Number of retry attempts to download RPMs")
+	buildFormatBumpCmd.Flags().IntVar(&buildFlags.downloadRetries, "retries", retriesDefault, "Number of retry attempts to download RPMs")
+	buildFormatOldCmd.Flags().IntVar(&buildFlags.downloadRetries, "retries", retriesDefault, "Number of retry attempts to download RPMs")
+	buildUpstreamFormatCmd.Flags().IntVar(&buildFlags.downloadRetries, "retries", retriesDefault, "Number of retry attempts to download RPMs")
```

Comment:
Shouldn't it be a top level flag instead like bundle-workers?

Reply:
There is already a bundle-workers flag that specifies the number of parallel workers when building bundles

Reply:
I know, I used it as an exemple of a top level flag.

Reply:
I'm suggesting that you replace all this code by:
`buildCmd.PersiustentFlags().IntVar(&buildFlags.downloadRetries, "retries", retriesDefault, "Number of retry attempts to download RPMs")`

Reply:
Oh, I see what you mean now... I can do that. It just seems weird to me that the option would be exposed to ~3 commands that don't use it.

Reply:
Updated the PR to make it a persistent flag